### ISSUE TITLE
Wiki needed

### ISSUE THREAD

**Issue by lovedheart** [giaf/hpmpc#3]
Hi. I have much interest in this implementation. But it's better to have a wiki about HPMPC. Thanks

**Comment by giaf** [comment]
Thanks for the interest. I am a bit busy at the moment writing my thesis, the wiki will come asap. Are you interested in control?

**Comment by lovedheart** [comment]
Yeah, I'm a student studying control theory. I'm doing a project which compares the different opensource qp solver and I wanna to choose one for solving QP problem of large-scale system. Is HPMPC suitable for that?.

**Comment by giaf** [comment]
It depends on what you mean by large-scale. HPMPC is optimized for dense small-scale MPC problems, and the performance is close to optimal for dense problems with up to a couple of hundreds states and controls.

**Comment by lovedheart** [comment]
I mean 1000-100000 variables.

**Comment by giaf** [comment]
You can have a try, but this is not certainly what it has been designed for.

**Comment by lovedheart** [comment]
Hi, how do you solve the linear system equations? LDL factorization or something else?

**Comment by giaf** [comment]
The solvers in HPMPC are designed to solve linear MPC problems, and not generic QPs. Therefore the (structured) KKT system of the unconstrained MPC problem is solved using a specialized Riccati recursion.
